components/director/internal/domain/runtime/service_test.go

```diff
@@ -226,6 +221,7 @@ func TestService_CreateWithMandatoryLabels(t *testing.T) {
 			},
 			EngineServiceFn: func() *automock.ScenarioAssignmentEngine {
 				svc := &automock.ScenarioAssignmentEngine{}
+				svc.On("AssignFormation", mock.Anything, tnt, runtimeID, graphql.FormationObjectTypeRuntime, model.Formation{Name: model.DefaultScenario}).Return(&model.Formation{Name: model.DefaultScenario}, nil)
```

Comment:
You could use the same `Context` as the test case for it to be more precise in the mocking, as the tests use `ctxWithSubaccount`, `ctx` and others.

Reply:
Not applicable for all tests, but I will change it where it's possible.